convert2rhel/checks.py

```diff
@@ -318,3 +297,26 @@ def get_unsupported_kmods(host_kmods, rhel_supported_kmods):
     Ignore certain kmods mentioned in the system configs. These kernel modules moved to kernel core, meaning that the
     functionality is retained and we would be incorrectly saying that the modules are not supported in RHEL."""
     return host_kmods - rhel_supported_kmods - set(system_info.kmods_to_ignore)
+
+
+def check_rhel_compatible_kernel_is_used():
+    """Check if the kernel packages are RHEL compatible."""
+    kernel_release, _ = run_subprocess("uname -r")
+    kernel_release = kernel_release.strip()
+    if not RHEL_COMPATIBLE_KERNEL_RELEASE_RE.match(kernel_release):
+        logger.critical(
+            "The kernel release {0} is not compatible "
+            "with the target RHEL system.\n\n"
+            "In order to convert, it is necessary to use Red Hat compatible "
+            "kernel (RHCK).\n\nTo install RHCK follow next steps:\n"
```

Comment:
```suggestion
            "The booted kernel {0} is not compatible "
            "with the target RHEL system.\n"
            "To proceed with the conversion, it is necessary to boot a RHEL-compatible "
            "kernel (RHCK).\n\nTo do that, execute the following steps:\n"
```
@mportman12, could you please help us to refine the message?

Reply:
"The booted kernel is not compatible with the target RHEL system. To proceed with the conversion, boot a RHEL-compatible kernel (RHCK) by executing the following steps:"

Reply:
thanks Miriam @mportman12